src/plugins/guided_onboarding/public/constants/guides_config/security.ts

```diff
@@ -53,11 +46,7 @@ export const securityConfig: GuideConfig = {
     {
       id: 'alertsCases',
       title: 'Alerts and cases',
-      descriptionList: [
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
-        'Nullam ligula enim, malesuada a finibus vel, cursus sed risus.',
-        'Vivamus pretium, elit dictum lacinia aliquet, libero nibh dictum enim, a rhoncus leo magna in sapien.',
-      ],
+      descriptionList: ['View and triage alerts.', 'Create cases.'],
```

Comment:
```suggestion
      descriptionList: ['View and triage alerts.', 'Create a case.'],
```

The plural on the case just doesn't make sense here. They're only going to open one case.